pymatgen/io/openmm/utils.py

```diff
@@ -341,130 +365,247 @@ def assign_charges_to_mols(
         charged_mols.append(openff_mol)
         # return a warning if some partial charges were not matched to any mol_xyz
     for unmatched_mol in inferred_mols - matched_mols:
-        warnings.warn(f"{unmatched_mol} in partial_charges is not isomorphic to any SMILE in the system.")
+        warnings.warn(
+            f"{unmatched_mol} in partial_charges is not isomorphic to any SMILE in the system.")
     return charged_mols
 
 
 def parameterize_system(
-    topology: Topology,
-    smile_strings: List[str],
-    box: List[float],
-    force_field: Union[str, Dict[str, str]] = "sage",
-    partial_charge_method: str = "am1bcc",
-    partial_charge_scaling: Dict[str, float] = None,
-    partial_charges: List[Tuple[Union[pymatgen.core.Molecule, str, Path], np.ndarray]] = [],
+        topology: Topology,
+        smile_strings: List[str],
+        box: List[float],
+        force_field: Union[str, Dict[str, str]] = "sage",
+        partial_charge_method: str = "am1bcc",
+        partial_charge_scaling: Dict[str, float] = None,
+        partial_charges: List[
+            Tuple[Union[pymatgen.core.Molecule, str, Path], np.ndarray]] = [],
 ) -> openmm.System:
     """
     Parameterize an OpenMM system.
 
     Args:
         topology: an OpenMM topology.
         smile_strings: a list of SMILEs representing each molecule in the system.
-        box: list of [xlo, ylo, zlo, xhi, yhi, zhi].
-        force_field: name of the force field. Currently only Sage is supported.
-        partial_charge_method:
-        partial_charge_scaling:
-        partial_charges:
+        box: list of [xlo, ylo, zlo, xhi, yhi, zhi] in nanometers.
+        force_field: Name of the force field or dict of forcefields for each
+                    small molecule, e.g. {"O": "spce"}. Small molecule
+                    forcefields and water models can either be provided
+                    informally, i.e. "gaff" or "sage" for small molecules,
+                    or "spce", "tip3p", or "tip4p" for water, or can be
+                    formally defined with OpenMM filenames,
+                    e.g. "charmm36/water.xml". Large molecule forcefields
+                    must be specified with the full path,
+                    e.g. "amber14/protein.ff14SB.xml".
+        partial_charge_method: Method for OpenFF partial charge assignment
+                                for small molecules without charges provided
+                                in partial_charges
+        partial_charge_scaling: Scaling for partial charges, either
```

Comment:
this actually cannot be done uniformly as a float, otherwise this is right

Reply:
fixed docstring